aot/src/test/java/com/dylibso/chicory/approvals/PackageSettings.java

```diff
@@ -0,0 +1,6 @@
+package com.dylibso.chicory.approvals;
+
+public class PackageSettings {
+    public String UseApprovalSubdirectory = "approvals";
```

Comment:
We have the tests under `chicory/approvals` with the results in `chicory/approvals/approvals` which seems redundant. I can see putting the results in a subdirectory if we didn't already use an `approvals` package for the tests.